### DISCUSSION TITLE
How to authenticate to clone beta images?

### DISCUSSION THREAD

**Discussion by tethridge** [cirruslabs/tart#850] [General]
I'd like to clone the macos-sequoia-xcode image, but I'm getting an AuthFailed error.  I have an Apple developer account, but I'm not sure how to authenticate to grab that pre-built image.
```
tart clone ghcr.io/cirruslabs/macos-sequoia-xcode:16-beta sequoia-xcode-16-beta
pulling manifest...
Error: AuthFailed(why: "received unexpected HTTP status code 403 while retrieving an authentication token", details: "{\"errors\":[{\"code\":\"DENIED\",\"message\":\"requested access to the resource is denied\"}]}\n")
```
Can somebody provide more details or point me to the documentation that explains this functionality?

**Comment by fkorotkov** [comment]
We don't yet have `sequoia` images with Xcodes. We'd like to build them on Sequoia host to make Apple ID login to work but unfortunately there are some hiccups.
